src/omen/cogs/wvw_commands.py

```diff
@@ -226,13 +228,15 @@ async def matchup(self, ctx, member: discord.Member = None):
                 embed_skirmish_value_string += f":{server}_circle: Points: {self.match_data_dict[server]['skirmish']['total_points']:=2} (+{self.match_data_dict[server]['skirmish']['ppt']:=2} per tick)\n"
             embed.add_field(name="Current Skirmish", value=embed_skirmish_value_string, inline=False)
             embed.add_field(name="Objectives", value=embed_obj_value_string, inline=False)
-            await ctx.channel.send(files=[author_img_attached, thumb_img_attached], embed=embed)
+            await interaction.response.send_message(files=[author_img_attached, thumb_img_attached], embed=embed)
         except Exception as e:
             print(e)
 
-    @commands.command()
-    async def worldpop(self, ctx, region: to_lower="na", member: discord.Member = None):
+    @app_commands.command(name="worldpop", description="Gets the world population")
+    async def worldpop(self, interaction: discord.Interaction, region: str):
```

Comment:
add validation for region. 

maybe something like 
if region.lower() != "eu" || region.lower() != "na" 
     return await interaction.response.send_message("error")

Reply:
added options for command + validation